src/TableContent.js

```diff
@@ -3,10 +3,40 @@ import PropTypes from 'prop-types'
 import { Table } from 'evergreen-ui'
 import URI from 'urijs'
 import ReactTooltip from 'react-tooltip'
+import _ from 'lodash'
 
 import tableHeaderCells from './tableHeaderCells'
 import TooltipIcon from './TooltipIcon'
 
+const fetchJson = async (endpoint, host, queryParams) => {
+  const url = URI(endpoint, host).search(queryParams).toString()
+  try {
+    const response = await fetch(url)
+    if (!response.ok) {
+      throw new Error(`${url} => ${response.status}`)
+    }
+    return await response.json()
+  } catch (e) {
+    return {
+      error: `${e.name}: ${e.message}`,
+      loading: false
+    }
+  }
+}
+
+const alertInvalidFiles = async (endpoint, host, checkedRows) => {
+  const response = await fetchJson(endpoint, host, {accession: checkedRows})
```

Comment:
Notice that if `fetchJson` throws an error it returns an object with an `error` and `loading` field. I guess you copied and pasted this from the React fetch loader. Here this return value doesn’t make sense. You should remove it and let any errors bubble up instead of catching them within `fetchJson`. Surround this block with `try` and `catch` and do what you think is best with the error: show it in an alert or log it to the console.